src/fsharp/CompileOps.fs

```diff
@@ -610,7 +610,8 @@ let getErrorString key = SR.GetString key
 let (|InvalidArgument|_|) (exn:exn) = match exn with :? ArgumentException as e -> Some e.Message | _ -> None
 
 let OutputPhasedErrorR errorStyle (os:System.Text.StringBuilder) (err:PhasedDiagnostic) =
-    let rec OutputExceptionR (os:System.Text.StringBuilder) = function        
+    let rec OutputExceptionR (os:System.Text.StringBuilder) error = 
```

Comment:
System.Text is opened at the top of the file:  https://github.com/Microsoft/visualfsharp/blob/master/src/fsharp/CompileOps.fs#L7